jobs/haproxy/spec

```diff
@@ -677,7 +677,9 @@ properties:
   ha_proxy.backend_health_rise:
     description: Number of consecutive successful health checks required before the server is considered healthy from an unhealthy state. The default value of 2 matches the default if the parameter is undefined. This parameter will be ignored if ha_proxy.backend_use_http_health is false.
     default: 2
-
+  ha_proxy.backend_https_check:
+    description: Set to true if the backend uses TLS on the health endpoint. Adds the check-ssl option to the backend configs. If backend certificate on traffic port is verified the Health endpoint cert will also be verified.  
+    default: false
```

Comment:
Could you please re-introduce an empty new-line after the health-check section?

Reply:
Done